rp_redcap/tests/test_views.py

```diff
@@ -20,6 +23,24 @@ def setUp(self):
         self.token = token.key
         self.client.credentials(HTTP_AUTHORIZATION="Token " + self.token)
 
+        self.org = self.create_org()
+        self.project = self.create_project(self.org)
+
+    def create_project(self, org):
```

Comment:
I think let's see if we could refactor this into a mixin test class, so that we don't have these function repeated in `test_views` and `test_tasks`